### ISSUE TITLE
Create onboarding / alpha disclaimer for new users.

### ISSUE THREAD

**Issue by CalebKester** [Playing-Indoors/KDM_WWW#90]
Need to display a message for first timers

**Comment by CalebKester** [comment]
@toconnell would this be a good use of your `/user/set_preferences/<user_id>` call would be to set a flag the first time they login so I can show them a welcome screen? I thought about just showing it if they don't have a settlement but then if they get invited to a settlement this wouldn't work.

So what about I just set a preference like "notFirstTimer: true" (name pending)

**Comment by toconnell** [comment]
@CalebKester I think...from an API perspective, this wants to be a dictionary/hash that includes boolean "flags" for various types of stuff. 

Kind of how the Survivor and Settlement objects have the `meta` element that contains a bunch of misc. key/value pairs, I think we want to have a lookup on the User called `notifications` or something that we use whenever we want to do a check like this.

Basically, for now, we do something like, `user: {notifications: {firstTimeAlpha: false, cookieWarning: false}}`

My thinking is that this is more extensible/reusable and doesn't junk up the user with a bunch of random-looking key/value elements. 

What do you think?

**Comment by CalebKester** [comment]
@toconnell I think that sounds pretty reasonable.
